posthog/api/test/test_exports.py

```diff
@@ -60,15 +64,21 @@ def setUpTestData(cls):
             created_by=cls.user,
             name="example insight",
         )
+        cls.tile = DashboardTile.objects.create(dashboard=cls.dashboard, insight=cls.insight)
         cls.exported_asset = ExportedAsset.objects.create(
             team=cls.team, dashboard_id=cls.dashboard.id, export_format="image/png", created_by=cls.user
         )
 
+    @patch("posthog.tasks.exports.image_exporter._export_to_png")
     @patch("posthog.api.exports.exporter")
-    def test_can_create_new_valid_export_dashboard(self, mock_exporter_task) -> None:
+    def test_can_create_new_valid_export_dashboard(self, mock_exporter_task, mock_export_to_png) -> None:
+        # add filter to dashboard
+        self.dashboard.filters = {"properties": [{"key": "$browser_version", "value": "1.0"}]}
+        self.dashboard.save()
+
         response = self.client.post(
             f"/api/projects/{self.team.id}/exports",
-            {"export_format": "image/png", "dashboard": self.dashboard.id},
+            {"export_format": "image/png", "dashboard": self.dashboard.id, "insight": self.insight.id},
```

Comment:
Doesn't this change the meaning of the test to `test_can_create_new_valid_export_insight`? (I might be missing something)

Reply:
Good point... this is a leftover PR from a few weeks ago. I'm not sure anymore. Looks like I mixed up what test to edit. Fixed now.